web/tests/narrow_unread.test.js

```diff
@@ -64,6 +65,7 @@ run_test("get_unread_ids", () => {
         id: 101,
         type: "stream",
         stream_id: sub.stream_id,
+        display_recipient: sub.name,
```

Comment:
Had to add this because of `assert(typeof message.display_recipient === "string");` in `adjusted_terms_if_moved`